### PR TITLE
Reorder EL1, expand remark

### LINKED ISSUE #295: EL1 Tweaks
- Not sure [Definition 5.1.9](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL1.html#def-exp-growvsdec) is a definition--what are we defining here?  Grow/Decay?  Feels more like an observation as stated.  If we really want this to be a definition, use `<term>`.
- [Remark 5.1.14](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL1.html#EL1-3-15) What do we mean by "growing at a continuous rate" here?  I think I'd say $2^x$ also grows at a continuous rate.  I'm guessing we are alluding to continuous vs. discrete compounding here.
- [Definition 5.1.15](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL1.html#def-number-e) isn't actually defining $e$.  

I don't know how much we want to get into motivating $e$ here, or if that should be an appendix, or what. Active Prelude devotes a [whole section](https://activecalculus.org/prelude/sec-exp-e.html) to this.   My general inclination is that nothing should be unmotivated, but this doesn't quite feel like the place.  Maybe we could expand [Activity 5.7.9](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL7.html#compound-interest) to continuous compounding to motivate the definition $e=\lim _{n \rightarrow \infty} \left(1+\frac{1}{n}\right)^n$ (if I remember right, this is actually the application that motivated Bernoulli to consider $e$ in the first place).

### PR DESCRIPTION
**Pull request by siwelwerd**
Closes #295.

### CODE REVIEW

**Comment by StevenClontz** [comment]
```
2024-07-02T17:39:11.9012343Z [31merror: [0m* PTX:ERROR:   a cross-reference ("xref") uses references [activity-motivate-e] that do not point to any target, or perhaps point to multiple targets.  Maybe you typed an @xml:id value wrong, maybe the target of the @xml:id is nonexistent, or maybe you temporarily removed the target from your source, or maybe an auxiliary file contains a duplicate.  Your output will contain some placeholder text that you will not want to distribute to your readers.
2024-07-02T17:39:11.9015808Z *              located within: "remark-number-e" (xml:id)
2024-07-02T17:39:13.0403147Z [31merror: [0m* PTX:ERROR:   a cross-reference ("xref") uses references [activity-motivate-e] that do not point to any target, or perhaps point to multiple targets.  Maybe you typed an @xml:id value wrong, maybe the target of the @xml:id is nonexistent, or maybe you temporarily removed the target from your source, or maybe an auxiliary file contains a duplicate.  Your output will contain some placeholder text that you will not want to distribute to your readers.
```

File: source/05-EL/01.ptx [hunk 1]
```diff
@@ -953,6 +898,65 @@
...
+    <remark xml:id="remark-number-e">
+      <statement>
+        <p>
+          Similar to how <m>\pi</m> arises naturally in geometry, there is an irrational number called <term><m>e</m></term> that
+          arises naturally when working with exponentials.  We usually use the approximation  <m>e \approx 2.718282</m>.  
+          <m>e</m> is also found on most scientific and graphing calculators. 
+        </p> 
+      </statement> 
+      <aside>
+        <p> See <xref ref="activity-motivate-e"/> for an exploration of how <m>e</m> arises naturally.
```
Comment (StevenClontz) [inline source/05-EL/01.ptx:911]:
```suggestion
        <p> See <xref ref="act-evaluate-base-e"/> for an exploration of how <m>e</m> arises naturally.
```

perhaps?

Comment (siwelwerd) [reply source/05-EL/01.ptx:911]: No, this was intentional, I guess this is blocked by #314

**Comment by siwelwerd** [comment]
Probably I screwed up the right workflow here. But the intent was for @tdegeorge to write an activity in 5.7, and then this reference that.  Could perhaps just remove the xref here, and note in #314 to go back and add it back in.

File: source/05-EL/01.ptx (hunk 1 above)
Comment (StevenClontz) [reply source/05-EL/01.ptx:911]: Yeah, we got our wish a couple weeks ago that CI/CD will be more agressive in blocking errors, such as non-existent xml-ids.

I think the solution then is to put a stub activity with that ID into this PR.

Comment (siwelwerd) [inline source/05-EL/01.ptx:911]:
```suggestion
        <p> See <xref ref="EL7"/> for an exploration of how <m>e</m> arises naturally.
```

**Comment by StevenClontz** [comment]
referencing the suggestion is a clean solution as well.

**Review by kathypinzon — APPROVED** [review]
this looks great!

**Pull request merged**